scripts/release.sh

```diff
@@ -87,6 +87,8 @@ CORE_APPS=( kernel odk couchdb-sync ui )
 CORE_COMPOSE='docker-compose.yml'
 CONNECT_APPS=( producer )
 CONNECT_COMPOSE='docker-compose-connect.yml'
+TEST_APPS=( integration-test )
+TEST_COMPOSE='docker-compose-test.yml'
```

Comment:
Maybe it's time to create the `docker-compose-release.yml` file with the containers to push to dockerhub.

What do you think?

Reply:
So a single compose inheriting the definitions from the other files? I don't anything that contains a `depends_on:` field can't be inherited from. Any other ideas?

Reply:
Yeah, boo: `ERROR: Cannot extend service 'kernel' in docker-compose.yml: services with 'depends_on' cannot be extended`

Reply:
Use `docker-compose-base.yml` instead?

Reply:
:drooling_face: Yep!